mne/label.py

```diff
@@ -2666,3 +2669,49 @@ def select_sources(subject, label, location='center', extent=0.,
                           hemi=new_label.hemi, name=name, subject=subject)
 
     return new_label
+
+
+def find_label_in_annot(pos, subject='fsaverage', annot='aparc.a2005s+aseg',
+                        subjects_dir=None):
+    """
+    Find name in atlas for given MRI coordinates.
+
+    Parameters
+    ----------
+    pos : ndarray, shape (3,)
+        Vector of x,y,z coordinates in MRI space.
+    subject : str
+        MRI subject name.
+    annot : str
+        MRI parcellation file name. Do not include the ``.mgz`` suffix.
+    subjects_dir : path-like
+        Path to MRI subjects directory.
+
+    Returns
+    -------
+    label : str
+        Anatomical region name from atlas.
+
+    Notes
+    -----
+    .. versionadded:: 0.24
+    """
+    import nibabel
```

Comment:
Give our standard error message:
```suggestion
    nibabel = _import_nibabel('read MRI parcellations')
```
and above import `_import_nibabel` from `.utils`